contrib/create-waku/src/renderTemplate.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+export function renderTemplate(src: string, dest: string) {
+  const stats = fs.statSync(src);
+
+  if (stats.isDirectory()) {
+    fs.mkdirSync(dest, { recursive: true });
+    for (const file of fs.readdirSync(src)) {
+      renderTemplate(path.resolve(src, file), path.resolve(dest, file));
+    }
+    return;
+  }
+
+  const filename = path.basename(src);
+
+  if (filename === 'package.json' && fs.existsSync(dest)) {
```

Comment:
Can you explain again when this happens?

Reply:
Maybe ``fs.existsSync(dest)``is not much relevant here.